### DISCUSSION TITLE
Possible to aggregate Kubernetes metrics in an on-prem installation (no Netdata Cloud account)?

### DISCUSSION THREAD

**Discussion by grandich** [netdata/netdata#18615] [Q&A]
Hi,

Can an on-prem netdata-parent installation with no associated Netdata Cloud account expose aggregated cluster and node metrics as detailed here?:  https://learn.netdata.cloud/docs/developer-and-contributor-corner/kubernetes-monitoring-with-netdata-overview-and-visualizations#cluster-and-node-metrics  (_The gauges and time-series charts you see right away in the Overview show aggregated metrics from every node in your cluster. For example, the apps.cpu chart (in the Applications menu item), visualizes the CPU utilization of various applications/services running on each of the nodes in your cluster._ )

**Comment by ilyam8** [comment]
Yes.
